src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -234,7 +234,7 @@ public WebAssemblyHost Build()
             // to configure services inside *that scope* inside their startup code, we create *both* the
             // service provider and the scope here.
             var services = _createServiceProvider();
-            var scope = services.GetRequiredService<IServiceScopeFactory>().CreateScope();
+            var scope = services.GetRequiredService<IServiceScopeFactory>().CreateAsyncScope();
 
             return new WebAssemblyHost(services, scope, Configuration, RootComponents, _persistedState);
```

Comment:
Is `WebAssemblyHost` `IAsyncDisposable`?

Reply:
Yes it looks like it: https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHost.cs#L22

Reply:
Then you need to change this https://github.com/dotnet/aspnetcore/blob/2105ef1ceb87dcd1d7bf6afbab9b3e6868ccd3bf/src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHost.cs#L24 to an `AsyncServiceScope`.

Reply:
Did you miss pushing an update with this change?

Reply:
Oops yeah I missed that, thanks, will update this file now

Reply:
Hrm no the file looks updated actually, https://github.com/dotnet/aspnetcore/blob/haok/asyncscope/src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHost.cs#L24  assuming I'm looking at the right place

Reply:
Ooops, might have missed it.